engines/decidim_hospitalet-surveys/app/commands/decidim_hospitalet/surveys/admin/create_survey_result.rb

```diff
@@ -89,6 +89,13 @@ def existing_user
           ).first
         end
 
+        def default_user
+          Decidim::User.where(
```

Comment:
Ok, that's a nice solution, although I still prefer mine xDDD. The advantage of going with my approach is that we'd be treating *stateful data* as *stateless application code*, which means that it'd transparently work on tests, etc.

But the whole solution is imperfect and tricky at best. Maybe we should consider creating an `anonymous` flag if this use case is more common in other installations, which would render the anonymous user useless.

So ok, let's go for this!